UNRELEASED_CHANGELOG.md

```diff
@@ -64,6 +64,9 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Now you can configure the style of `AttachmentMediaActivity`
+- Added `streamUiLoadingView`, `streamUiEmptyStateView` and `streamUiLoadingMoreView` attributes to `ChannelListView` and `ChannelListViewStyle`
+- Added possibility to customize `ChannelListView` using theme. Check `StreamUi.ChannelListView` style.
```

Comment:
The user needs to override `streamUiChannelListViewStyle`. I think it is worth make this clear.